src/dynamicprompts/parser/parse.py

```diff
@@ -15,6 +17,15 @@
 )
 from dynamicprompts.parser.action_builder import ActionBuilder
 
+
+@dataclass
```

Comment:
```suggestion
@dataclass(frozen=True)
```
so we can trust `default_parser_config` to be immutable.

Reply:
Agreed